_posts/2020-03-18-cockpit-215.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Cockpit 215
+author: kkoukiou
+date: 2020-03-18
+tags: cockpit linux
+slug: cockpit-215
+category: release
+summary: Cockpit with Network improvements and Docker deprecation
+comments: 'true'
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/). We release regularly. Here are the release notes from version 215.
+
+### Networking: Show additional ports for each firewall zone
+
+The Firewall page now shows ports that are directly added to a zone right after the services list.
+![zone ports](/images/firewall-zone-ports.png)
+
+### Docker: Deprecate in current developement series
```

Comment:
```suggestion
### Docker: Deprecate and cease development
```

Reply:
This is not the same.

Reply:
"Docker: Deprecate in current developement series" isn't correct either, and doesn't say what a series is or anything.

How's this for a 1-line summary instead then?

```suggestion
### Docker: Deprecate `cockpit-docker` in favor of `cockpit-podman`
```

And we can explain it more below the heading. (Which we do.)

Reply:
I am ok with this, @martinpitt ?

Reply:
I like that suggestion.